clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtilSpec.groovy

```diff
@@ -416,4 +414,39 @@ class GCEUtilSpec extends Specification {
       9              | 5              | 7              | 7
       6              | 5              | 7              | 6
   }
+
+  @Unroll
+  void "checkAllForwardingRulesExist should fail only if we can't find all loadbalancers"() {
```

Comment:
I think this description could be a bit more precise. It should fail if _any_ of the requested load balancers aren't found, right? Not only if _all_ are not found.